com.unity.perception/CHANGELOG.md

```diff
@@ -23,7 +23,9 @@ Added a `projection` field in the capture.sensor metadata. Values are either "pe
 
 ### Changed
 
-Changed the JSON serialization key of Normal Sampler's standard deviation property from "standardDeviation" to "stddev". Scneario JSON configurations that were generated using previous versions will need to be manually updated to reflect this change.
+Scenario JSON configurations that were generated using previous versions will need to be manually updated to reflect the following changes:
```

Comment:
We usually keep this type of call out for the "upgrade notes" part of release notes when a new version of the package is released. I think for the changelog it might be better to keep the language consistent with previous entries?